include/tvm/topi/transform.h

```diff
@@ -321,6 +321,11 @@ inline Tensor reshape(const Tensor& x, Array<PrimExpr> newshape, std::string nam
   auto x_shape = x->shape;
   Array<PrimExpr> target_shape;
 
+  // if newshape is an empty tensor make it [0] instead.
+  if (newshape.empty()) {
+    newshape.push_back(0);
+  }
```

Comment:
I think we decided this snippet is actually not needed. Can you remove it and confirm everything still works?

Reply:
:+1: